src/api/config.go

```diff
@@ -1,14 +1,20 @@
 package api
 
 import (
-	"fmt"
 	"github.com/go-resty/resty/v2"
+	"github.com/spf13/cast"
 	"keboola-as-code/src/client"
-	"keboola-as-code/src/json"
 	"keboola-as-code/src/model"
-	"strconv"
 )
 
+func (a *StorageApi) ListComponents(branchId int) (*[]*model.Component, error) {
```

Comment:
Presunute do `config.go`, kedze to tiez nacitava konfiguracie -> ale pre vsetky komponenty.